designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputDropDown.kt

```diff
@@ -33,22 +54,165 @@ import org.hisp.dhis.mobile.ui.designsystem.theme.TextColor
  * item is focused
  * @param modifier allows a modifier to be passed externally
  * @param onResetButtonClicked callback to when reset button is clicked
- * @param onArrowDropDownButtonClicked callback to when arrow drop down button is clicked
+ * @param showSearchBar config whether to show search bar in the bottom sheet
  */
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun InputDropDown(
     title: String,
     state: InputShellState,
-    selectedItem: String? = null,
+    dropdownItems: List<DropdownItem>,
+    selectedItem: DropdownItem? = null,
     supportingTextData: List<SupportingTextData>? = null,
     legendData: LegendData? = null,
     isRequiredField: Boolean = false,
     modifier: Modifier = Modifier,
     onFocusChanged: ((Boolean) -> Unit)? = null,
     onResetButtonClicked: () -> Unit,
-    onArrowDropDownButtonClicked: () -> Unit,
+    onItemSelected: (DropdownItem) -> Unit,
+    showSearchBar: Boolean = true,
```

Comment:
is this optional? I thought it was always visible for dropdown in the bottombar case

Reply:
Yep, Marcos wanted it to be configurable by the app whether to show the search or not.